tests/test_kubernetes.py

```
@@ -2315,3 +2323,201 @@ def test_close_cleanup_error(
 
         session.container_api.stop_container.assert_called_once()
+
+
+class TestKubernetesStreamingCallbacks:
+    """Test on_stdout/on_stderr callback support in Kubernetes execute_command."""
+
+    @patch("kubernetes.config.load_kube_config")
+    @patch("llm_sandbox.kubernetes.CoreV1Api")
+    @patch("llm_sandbox.language_handlers.factory.LanguageHandlerFactory.create_handler")
+    def test_execute_command_with_stdout_callback(
+        self, mock_create_handler: MagicMock, mock_core_v1_api: MagicMock, mock_load_config: MagicMock
```

Ruff (0.15.1)

[warning] Unused method argument: `mock_core_v1_api`

(ARG002)